nipoppy_cli/README.md

```diff
@@ -16,7 +16,7 @@ The framework includes three components:
     ![Nipoppy specification](img/nipoppy_specification.jpg)
 
 2. A protocol for data organization, curation and processing, with steps that include the following:
-    - **Conversion** of raw scanner output to BIDS
+    - **Organization** of raw data, including conversion of raw scanner output to [BIDS](https://bids.neuroimaging.io/)
```

Comment:
Let's just say DICOMs and now raw scanner output which can be several things..

Reply:
Added "DICOMs or NIfTIs"